[camerax] Loosen restrictions of fallback strategies for choosing resolutions (#5239)
flutter/packages: packages/camera/camera_android_camerax/CHANGELOG.md

## 0.5.0+21

> * Changes fallback resolution strategies for camera use cases to look for a higher resolution if first the desired
>   resolution is unavailable and then also a lower resolution is unavailable.

Reviewer: Completely optional reword, but I personally found the original phrasing confusing

```suggestion
* Changes fallback resolution strategies for camera use cases to look for a higher resolution if neither the desired
  resolution nor any lower resolutions are available.
```
